index.html

```diff
@@ -1,68 +1,67 @@
 <!DOCTYPE html>
 <html>
-              Here goes the filter menu.
-              -->
-          </form>
-        </section>
-        <section class="allMovies">
-          <figure class="movies">
-            <!--
-               Here goes all movies.
-               -->
-          </figure>
-        </section>
-        <section class="extraSection">
-          <!-- here goes all the cards like present card 4DX!...-->
-        </section>
-      </section>
-      <section class="lowerSection">
-        <figure class="uppKommingMovies">
-          <!--up coming movies here-->
-        </figure>
-        <div class="uppKommingEvents">
-          <!--All the events here-->
-        </div>
-      </section>
-    </main>
-    <section class="skiReviewerSection">
-      <figure>
-        <img
-          src="./pictures/skier.png"
-          alt="Skidåkare som ser på bio"
-          class="skiReviewer"
-        />
-        <blockquote>Oerhört mäktigt</blockquote>
-        <figcaption>-Stig Helmer</figcaption>
-      </figure>
-      <figure>
-        <img
-          src="./pictures/skierTwo.png"
-          alt="Skidåkare som ser på bio"
-          class="skiReviewer"
-        />
-        <blockquote>Helt underbar helg</blockquote>
-        <figcaption>-Johan Jonsson</figcaption>
-      </figure>
-      <figure>
-        <img
-          src="./pictures/skierThree.png"
-          alt="Skidåkare som ser på bio"
-          class="skiReviewer"
-        />
-        <blockquote>Fantastiskt!</blockquote>
-        <figcaption>-Frida Karlsson</figcaption>
-      </figure>
-    </section>
-    <footer>
-      <nav>
-        <a href="#" class="footerNav footerAboutUsLink">Om oss</a>
-        <a href="#" class="footerNav footerAllMoviesLink">Alla filmer</a>
-        <a href="#" class="footerNav footerInstagramLink">Instagram</a>
-        <a href="#" class="footerNav footerFacebookLink">Facebook</a>
-        <a href="#" class="footerNav footerXLink">X</a>
-        <a href="#" class="footerNav footerTosLink">ToS</a>
-        <a href="#" class="footerNav footerPrivacyPolicyLink">Privacy policy</a>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>Kino</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+  <script type="module" src="main.js" defer></script>
+  <script src="https://kit.fontawesome.com/8357c99e29.js" crossorigin="anonymous"></script>
+  <!--The script above needs to be added on every singel html file that contains moviecard-->
+</head>
+
```

Comment:
The main tag from index.html have dissapeared and a lot of section that should be present have also disappeared.